Modelica/Resources/Scripts/Conversion/ConvertModelica_from_3.2.3_to_4.0.0.mos

```diff
@@ -1761,6 +1761,15 @@ convertElement({"Modelica.Electrical.Machines.BasicMachines.SynchronousInduction
                 "Modelica.Electrical.Machines.BasicMachines.SynchronousInductionMachines.SM_ElectricalExcited",
                 "Modelica.Electrical.Machines.BasicMachines.SynchronousInductionMachines.SM_ReluctanceRotor"},
                 "airGapR", "airGap")
+convertModifiers({"Modelica.Electrical.Spice3.Internal.MOS2"},
+                  {"vp"}, fill("", 0), true)
```

Comment:
The `true` is kind of superfluous here I think.

Reply:
True. Stiil OK to accept?